impeller/entity/contents/content_context.cc

```diff
@@ -69,6 +70,7 @@ void ContentContextOptions::ApplyToPipelineDescriptor(
       color0.dst_color_blend_factor = BlendFactor::kOne;
       color0.src_alpha_blend_factor = BlendFactor::kZero;
       color0.src_color_blend_factor = BlendFactor::kZero;
+      color0.write_mask = ColorWriteMaskBits::kNone;
```

Comment:
I suppose it makes sense that this is correct. Have no clue why it could be faster. RSLGTM?